light/errors.go

```diff
@@ -66,28 +89,13 @@ func (e errNoWitnesses) Error() string {
 	return "no witnesses connected. please reset light client"
 }
 
-type badWitnessCode int
-
-const (
-	noResponse badWitnessCode = iota + 1
-	invalidLightBlock
-)
-
 // errBadWitness is returned when the witness either does not respond or
 // responds with an invalid header.
 type errBadWitness struct {
-	Reason       error
-	Code         badWitnessCode
-	WitnessIndex int
+	Reason error
+	Index  int
```

Comment:
WitnessIndex is more informative. Index by itself could mean 1000 different things

Reply:
Yeah you're right. I will change it